rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/180_locale_dependent_mapping.yml

```diff
@@ -16,26 +16,26 @@
                       date_field:
                           type: date
                           format: "8E, d MMM uuuu HH:mm:ss Z"
-                          locale: "de"
+                          locale: "fr"
```

Comment:
Note to self: the reason for choosing French is [here](https://github.com/elastic/elasticsearch/pull/110222/files#r1721775548).